masonry/src/event_loop_runner.rs

```diff
@@ -51,11 +54,10 @@ pub enum WindowState<'a> {
     },
 }
 
-struct MainState<'a> {
+pub struct MasonryState<'a> {
```

Comment:
We should have a doc comment here explaining how to integrate with an external `winit` event loop (i.e. call `::new`, and then pass through all events as needed.

Reply:
Ok, added.